package.json

```diff
@@ -11,7 +11,8 @@
     "test": "nps test",
     "test:ts": "(tsc -p ./tsconfig.json && rimraf test-ts) || rimraf test-ts",
     "commitmsg": "opt --in commit-msg --exec \"validate-commit-msg\"",
-    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\""
+    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\"",
+    "doctoc": "doctoc --maxlevel 3 --title '**Table of Contents**' README.md"
```

Comment:
Super! In this project, we use `nps` which puts scripts in `package-scripts.js` Could you move this script to there? Then you'll run `npm start doctoc`. That way we can add a helpful `description` :+1: